examples/react/start-neon-basic/src/routes/handler.$.tsx

```
@@ -0,0 +1,25 @@
+import { StackHandler } from "@stackframe/react";
+import { stackClientApp } from "../stack";
```

ESLint

[error] `../stack` import should occur after import of `react`

(import/order)